### DISCUSSION TITLE
Tray indicators have TOO SMALL borders, and are very difficult to see !

### DISCUSSION THREAD

**Discussion by MichealousFabulosos** [jonaskohl/CapsLockIndicator#184] [Ideas]
Hello !
We need to be able to configure the tray icons indicators because the difference between state ON or OFF is to small to see !
The status is only indicated by a thin border.
Please improve this!
Thanks.
![Capture d’écran ON](https://github.com/jonaskohl/CapsLockIndicator/assets/158620859/487004aa-b83d-49df-a635-972797987cd6)
![Capture d’écran OFF](https://github.com/jonaskohl/CapsLockIndicator/assets/158620859/d7613471-9768-43fa-88d3-3a5cc9785484)

**Comment by JuanFe789** [comment]
I was thinking the same thing, too difficult to see, so I changed the Icon to other i like. Go to advance options > download icon pack. 

i´m using these right now:
<img width="58" height="29" alt="image" src="https://github.com/user-attachments/assets/0eef6c89-f372-4657-875d-c99e60f82131" />
<img width="50" height="30" alt="image" src="https://github.com/user-attachments/assets/e56dbcf6-99e2-4d11-8af1-31d18a01f2fb" />
But you can chose whatever you like

**Reply by MichealousFabulosos** [reply]
I was talking about visibility of Enabled / Disabled of each features. ;)
